core/storage/suggestion/gae_models.py

```diff
@@ -512,6 +517,11 @@ def get_voiceover_applications(cls, target_type, target_id, language_code):
             cls.target_type == target_type, cls.target_id == target_id,
             cls.language_code == language_code)).fetch()
 
+    @staticmethod
+    def get_export_policy():
+        """Model does not contain user data."""
```

Comment:
It does, doesn't it? There are author_id fields etc.

Please write a takeout method for this and update the design doc accordingly.

Reply:
Updated.